### DISCUSSION TITLE
Multiple accounts feature

### DISCUSSION THREAD

**Discussion by MaxxStarr** [mahmoudbahaa/outlook-for-linux#4] [Ideas]
Hi, in windows version of outlook app you can also add other e-mail accounts besides outlook. it would be very cool and useful if i could use this app both for my outlook and gmail email :)

**Comment by leenozara** [comment]
I too would like to use the app for several accounts. I do this with the Teams app, from which this project is forked, by using the "customUserDir" parameter. This gives me a Teams app for each organization I'm a member. However, I can't seem to get this to work for this Outlook application; it seems no data persists and I'm forced to re-login each time the app is closed.

```Desktop Entry]
Name=Outlook Work
Exec=flatpak run io.github.mahmoudbahaa.outlook_for_linux --appTitle "Outlook Work" --customUserDir="/home/lee/.msoutlook/work"
Terminal=false
Type=Application
Icon=io.github.mahmoudbahaa.outlook_for_linux
StartupWMClass=outlook-for-linux
Comment=Unofficial Microsoft Outlook client for Linux using Electron. It uses the Web App and wraps it as a standalone application using Electron.
MimeType=x-scheme-handler/msoutlook;
Categories=Chat;Network;Office;
X-Desktop-File-Install-Version=0.27
X-Flatpak-RenamedFrom=outlook-for-linux.desktop;
```

**Comment by gj241** [comment]
This is now available in the maintained fork **outlook-for-linux v1.4.0**: https://github.com/gj241/outlook-for-linux/releases/tag/v1.4.0

There's a per-account switcher (a pill in the top-right, draggable) — each account runs on its own persistent partition, so you stay logged in to all of them and switch instantly (recently-used accounts are cached; others reload without re-signing-in). Add account, rename, and remove are all in the switcher. Installers (deb/AppImage/rpm) are on the release page.
